api/note.ts

```diff
@@ -78,14 +74,14 @@ async (args: {
     page: number;
     limit: number;
     placeId?: string;
-    sort?: NoteSortByEnum;
+    sort?: "recent" | "trending";
 }) => {
     const url = copyURL(urlArg);
     url.pathname = `/getNotes`;
     url.searchParams.set("page", String(args.page));
     url.searchParams.set("limit", String(args.limit));
     if (args.placeId) {
-        url.searchParams.set("placeId", String(args.placeId));
+        url.searchParams.set("place_id", String(args.placeId));
```

Comment:
Why is this modification necessary?

The `placeId` is used in the API.

You can find it here: https://github.com/satlantis-dev/api/blob/c3cca337685fde46c0a32ef2c501b5d1b6b41023/database/note.go#L382

Reply:
It should indeed use `placeId` here.

https://github.com/satlantis-dev/api/blob/c3cca337685fde46c0a32ef2c501b5d1b6b41023/rest/note.go#L341